### ISSUE TITLE
Prenatal double bubble sign

### ISSUE THREAD

**Issue by patt1morgan** [obophenotype/human-phenotype-ontology#6996]
**Preferred term label:** Prenatal double bubble sign

**Synonyms** double bubble sign, double bubble

**Definition (free text, please give PubMed ID)  Sonographic detection of a double bubble sign in the upper abdomen is strongly indicative of duodenal obstruction. One bubble represents fetal stomach, and the other is attributed to a dilated proximal part of the duodenum; continuity between both bubbles is required for the sign.
https://doi-org.ezproxy.cul.columbia.edu/10.1002/pd.4652                
PMID: 31167209
**Parent term (use hpo.jax.org/app)**Abnormality of prenatal development or birth HP:0001197

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**Duodenal atresia ORPHA:1203  Trisomy 21 OMIM# 190685, Heterotaxy syndrome
ORPHA:450**

Your nano-attribution (ORCID)**

**Comment by patt1morgan** [comment]
@rhiannonmellis, @jokemuys.  Thank you for agreeing to work with us to advance the perinatal section of the HPO. We would appreciate it if you could review this new term request and approve it or help us improve it. Any suggestions welcome!

**Comment by rhiannonmellis** [comment]
Hi @patt1morgan, great suggestion for a new term. I agree with the entry as written, nothing more to add!

**Comment by pnrobinson** [comment]
added thanks @patt1morgan
